website/docs/config-file.md

```diff
@@ -30,11 +30,17 @@ no-multigrid = false
 vsync = true
 maximized = false
 srgb = false
+fork = false
 idle = true
 neovim-bin = "/usr/bin/nvim" # in reality found dynamically on $PATH if unset
 frame = "full"
+theme = "auto"
 title-hidden = true
 tabs = true
```

Comment:
Could you sort this and the settings in `config.rs` alphabetically. That way the order is less random, and it's easier to compare.

Reply:
Done in ab188aa. Local `cargo test` also ran successfully after the change. Can never be too safe